src/py42/sdk/queries/query_filter.py

```diff
@@ -150,6 +163,10 @@ def on_same_day(cls, value):
             cls._term, formatted_start_time, formatted_end_time
         )
 
+    @classmethod
+    def within_the_period(cls, value):
```

Comment:
I think this should be `within_the_last` to match the language of the query itself. It also is more specific since  a "period" could be any two points in time, but "the last" is only from now backwards "x" units.

Reply:
This should also have a docstring.

Reply:
Changed. Added.